### PR TITLE
fix NFT load from chain

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

File: src/components/Donations/Confirmation/index.tsx [hunk 1]
```diff
@@ -2,8 +2,16 @@ import styled from 'styled-components'
 import { Icon } from '@components'
 import { rgba } from '@lib'
 import { palette } from '@theme'
+import { useContext, useEffect } from 'react'
+import DonationContext from 'src/lib/donationContext'
```
Comment (EricWVGG) [inline src/components/Donations/Confirmation/index.tsx:7]: could we update this export to…
```
import { DonationContext } from '@lib'
```

File: src/components/Sidebar/index.tsx [hunk 2]
```diff
@@ -29,26 +44,35 @@ const Sidebar = ({ FUND_COUNT, FUND_AMOUNT, FUND_GOAL, FUND_DATE, FUND_DEADLINE
...
+  const fundStart = raffle ? new Date(raffle?.startTime.toNumber() * 1000) : undefined
 
   return (
     <StyledStickyCanvas>
       <Wrapper>
         <Header>
-          Join our <strong>{FUND_DATE}</strong> donation drive
+          Join our{' '}
+          <strong>
+            {fundStart ? monthNames[fundStart?.getMonth()] : ''}, {fundStart?.getFullYear()}
```
Comment (EricWVGG) [inline src/components/Sidebar/index.tsx:60]: does `fundStart?.getMonth()` return 0 or 1 for January? (I have no idea, just double-checking)

File: src/pages/index.tsx [hunk 3]
```diff
@@ -10,16 +10,31 @@ import {
...
+import raffleAbi from 'src/contracts/RaffleAbi'
+import { useEffect } from 'react'
 
 const Home = () => {
-  const FUND_COUNT = 3.2
-  const FUND_AMOUNT = 15250
   const FUND_GOAL = 25000
-  const FUND_DATE = 'May, 2022'
-  const FUND_DEADLINE = '2022-06-30T00:00:00'
+  const raffleContractAddress = assert(
```
Comment (EricWVGG) [inline src/pages/index.tsx:21]: so happy to finally see these placeholders gone…

**Review by EricWVGG — APPROVED** [review]

File: src/components/Sidebar/index.tsx (hunk 2 above)
Comment (rodrigopavezi) [reply src/components/Sidebar/index.tsx:60]: Hey, good question. It returns 0

File: src/components/Donations/Confirmation/index.tsx (hunk 1 above)
Comment (rodrigopavezi) [reply src/components/Donations/Confirmation/index.tsx:7]: Good catch

**Pull request merged**